perfzero/lib/cloud_manager.py

```diff
@@ -160,34 +132,93 @@ def create(username, project, zone, machine_type, accelerator_count,
                  'instance:\n'
                  'git clone https://github.com/tensorflow/benchmarks.git\n'
                  'sudo usermod -a -G docker $USER\n')
-  else:
-    cmd = 'gcloud compute ssh {} --project={} --zone={} --command="git clone {}"'.format(  # pylint: disable=line-too-long
-        instance_name, project, zone,
-        'https://github.com/tensorflow/benchmarks.git')
-    run_command(cmd, is_from_user=True)
-    logging.info('Successfully checked-out PerfZero code on the '
-                 'computing instance\n')
-
-    cmd = 'gcloud compute ssh {} --project={} --zone={} --command="sudo usermod -a -G docker $USER"'.format(  # pylint: disable=line-too-long
-        instance_name, project, zone)
-    run_command(cmd, is_from_user=True)
-    logging.info('Successfully added user to the docker group\n')
-
-  cmd = 'gcloud compute ssh {} --project={} --zone={} -- -L 6006:127.0.0.1:6006'.format(  # pylint: disable=line-too-long
-      instance_name, project, zone)
+    return
+
+  remote_cmd = 'git clone https://github.com/tensorflow/benchmarks.git'
+  run_command(_ssh_cmd(instance_name, project, zone, remote_cmd),
+              is_from_user=True)
+  logging.info('Successfully checked-out PerfZero code on the '
+               'computing instance\n')
+
+  remote_cmd = 'sudo usermod -a -G docker $USER'
+  run_command(_ssh_cmd(instance_name, project, zone, remote_cmd),
+              is_from_user=True)
+  logging.info('Successfully added user to the docker group\n')
+
+  port_forwarding_cmd = (_ssh_prefix(instance_name, project, zone) +
+                         ' -- -L 6006:127.0.0.1:6006')
   logging.info('Run the command below to ssh to the instance together with '
-               'port forwarding for tensorboard:\n%s\n', cmd)
+               'port forwarding for tensorboard:\n%s\n', port_forwarding_cmd)
+
+
+def _instance_names(instance_name, num_instances):
```

Comment:
If user creates two instance, the name of the first in stance will be `instance_name-1`. If user creates one instance, the name of the first instance will be `instance_name`. These two names are different.

Let's say user executes `cloud_manager.py create --num_instances=1` followed by `cloud_manager.py create --num_instances=2`. Three instances will be created. It seems a bit counter-intuitive, since user probably expects two instances to be created with his/her name.